coretex/_task/_local.py

```diff
@@ -116,16 +136,25 @@ def _readTaskRunConfig() -> List[BaseParameter]:
             parameter = parameter_factory.create(parameterJson)
             parameters.append(parameter)
 
-    parameterValidationResults = validateParameters(parameters, verbose = True)
-    if not all(parameterValidationResults.values()):
-        # Using this to make the parameter errors more readable without scrolling through the console
-        sys.exit(1)
-
     return parameters
 
 
+def _getParsedParameterValue(
+    parameterName: str,
+    parser: Any,
+    default: Any
+) -> Optional[Any]:
+
+    parsedParameter = getattr(parser, parameterName)
+    if parsedParameter is None:
+        return default
+
+    return parsedParameter
+
+
 def processLocal(args: Optional[List[str]] = None) -> Tuple[int, TaskCallback]:
-    parser, unknown = LocalArgumentParser().parse_known_args(args)
+    parameters = _readTaskRunConfig()
```

Comment:
`_readTaskRunConfig` should also be moved to `LocalArgumentParser` as a `classmethod`